foreman/client.py

```diff
@@ -393,20 +394,24 @@ def _get_local_defs(self, strict=True):
             if f_ver.split('.')[:2] == self.version.split('.')[:2]:
                 last_major_match = f_name
             if ver_cmp(f_ver, self.version) < 0:
```

Comment:
It's simpler than that, just change the '<' for '>'.

The idea here is to stop looking if the major version of the file is higher than our major version, because we acknowledge that they are not backwards compatible (so 1.4 will not work on 1.3).